### PR TITLE
Updated Branching Strategy

### PR DESCRIPTION
**Pull request by aleoyakas**
Include details on:
- updating documentation
- merging non-functional changes

### CODE REVIEW

File: docs/6.-Engineering/Source-Control,-Versioning-&-Branching-Strategy/README.md [hunk 1]
```diff
@@ -25,9 +25,10 @@ Code is managed and stored using [Git](https://git-scm.com/docs) and GitHub.
 | **Branch** | **Key Purpose** | **Naming** |
 |--|--|--|
 | **Main** | Reflection of production environments. At any point you should be able to branch from Main absolutely confident that it will match what is  | `main` |
-| **Release** | A release branch is used when EITHER of following conditions are met: <br> 1. multiple features are being released together <br> 2. BAT is required. | `release/{Release-Name]` |
-| **Feature** | Contains all the changes required for a specific feature or bug, only. Note that the term "feature branch is used as it is common practice, but it is synonymous with a backlog item - an independent, releasable, valuable product increment. | `feature/{Feature-Name}` <br> OR <br> `bug/{Bug-Name}` |
+| **Release** | A release branch is used when EITHER of following conditions are met: <br> 1. multiple features are being released together <br> 2. BAT is required | `release/{Release-Name]` |
```
Comment (91dave) [inline docs/6.-Engineering/Source-Control,-Versioning-&-Branching-Strategy/README.md:28]: @aleoyakas - small thing but make the brackets consistent in the placeholder, i.e. `release/{Release-Name}`

**Review by 91dave — CHANGES_REQUESTED** [review]
See command about small typo.

Would also suggest maybe adding the new wording as a section, rather than a note?

**Review by edwardearle — APPROVED** [review]
Looks good. Appreciate the addition of the documentation category too. Thanks @aleoyakas

**Review by 91dave — APPROVED** [review]
Changes made as requested.

**Pull request merged**